c10/core/Allocator.cpp

```diff
@@ -16,10 +16,6 @@ at::DataPtr InefficientStdFunctionContext::makeDataPtr(
           device};
 }
 
-} // namespace c10
-
-namespace caffe2 {
-
 C10_API at::Allocator* allocator_array[static_cast<int>(
     at::DeviceType::COMPILE_TIME_MAX_DEVICE_TYPES)];
```

Comment:
we have already  casted constant: https://github.com/pytorch/pytorch/blob/master/c10/DeviceType.h#L33